dist/pom.xml

```diff
@@ -184,6 +184,102 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>generate-licenses-html</id>
+            <activation>
+                <property>
+                    <name>!skip-generate-licenses-html</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.7</version>
+                        <executions>
+                            <execution>
+                                <phase>prepare-package</phase>
+                                <configuration>
+                                    <tasks>
+                                        <fileset id="licenses.xml.list" dir="${basedir}/target/${server.output.dir.prefix}-${jboss.as.release.version}/docs/licenses" includes="*licenses.xml"/>
+                                        <!--
+                                        <property name="prop.licenses.xml.list" refid="licenses.xml.list"/>
+                                        <echo>List of licenses.xml in ${basedir}/target/${server.output.dir.prefix}-${jboss.as.release.version}/docs/licenses</echo>
+                                        <echo>${prop.licenses.xml.list}</echo>
+                                        -->
+                                        <pathconvert pathsep="&#xA;" property="license-files-list-items" refid="licenses.xml.list">
+                                            <chainedmapper>
+                                                <mapper type="flatten"/>
+                                                <globmapper from="*" to="&lt;item&gt;*"/>
+                                                <globmapper from="*" to="*&lt;/item&gt;"/>
+                                            </chainedmapper>
+                                        </pathconvert>
+                                        <echo file="${basedir}/target/${server.output.dir.prefix}-${jboss.as.release.version}/docs/licenses/licenses.xml">&lt;?xml version="1.0" encoding="UTF-8"?&gt;&#xA;&lt;list&gt;&#xA;</echo>
+                                        <echo file="${basedir}/target/${server.output.dir.prefix}-${jboss.as.release.version}/docs/licenses/licenses.xml" append="true">${license-files-list-items}&#xA;</echo>
+                                        <echo file="${basedir}/target/${server.output.dir.prefix}-${jboss.as.release.version}/docs/licenses/licenses.xml" append="true">&lt;/list&gt;</echo>
+                                    </tasks>
+                                </configuration>
```

Comment:
This part generates file which contains list of *licenses.xml files provisioned by feature packs (using ant task)